File: cloud/config-server/src/main/java/pers/acp/admin/config/ConfigServerApplication.java
package pers.acp.admin.config;

import org.springframework.boot.SpringApplication;
import org.springframework.cloud.config.server.EnableConfigServer;
import pers.acp.spring.cloud.annotation.AcpCloudAtomApplication;

/**
 * @author zhang by 27/02/2019
 * @since JDK 11
 */
@AcpCloudAtomApplication
@EnableConfigServer
public class ConfigServerApplication {

    public static void main(String[] args) {
        SpringApplication.run(ConfigServerApplication.class, args);
    }

}

File: cloud/config-server/src/main/kotlin/pers/acp/admin/config/ConfigServerApplication.kt
package pers.acp.admin.config

import org.springframework.boot.runApplication
import org.springframework.cloud.config.server.EnableConfigServer
import pers.acp.spring.cloud.annotation.AcpCloudAtomApplication

/**
 * @author zhang by 27/02/2019
 * @since JDK 11
 */
@AcpCloudAtomApplication
@EnableConfigServer
class ConfigServerApplication

fun main(args: Array<String>) {
    runApplication<ConfigServerApplication>(*args)
}
